edge/pkg/metamanager/metaserver/kubernetes/storage/sqlite/store.go

```diff
@@ -123,6 +123,10 @@ func (s *store) Count(key string) (int64, error) {
 	panic("implement me")
 }
 
+func (s *store) RequestWatchProgress(ctx context.Context) error {
```

Comment:
For Kubernetes v1.28, a new funtion RequestWatchProgress need to be added for storage interface. Do we need to achieve it in metaserver? @ZhengXinwei-F